test/bigtest/tests/PurchaseOrder/update-order-error-modal-test.js

```diff
@@ -25,7 +25,7 @@ describe('Update Order Error modal', () => {
         <UpdateOrderErrorModal
           cancel={cancelFake}
           orderNumber="123"
-          errorCode="vendorIsInactive"
+          errors={[{ code: 'vendorIsInactive' }]}
```

Comment:
It looks like a good place to use constant

Reply:
fixed